Added "Setting SLO targets", why not set 100% target (#11197)
DataDog/documentation: content/en/monitors/service_level_objectives/_index.md

3. Set up to three SLO targets. Each target consists of a target percentage and a rolling time window. Available time windows are: 7 days, 30 days, and 90 days. It is recommended that you make the SLO target percentage stricter than the target percentages stipulated in your SLAs.
4. Finally, give the SLO a title, describe it in more detail or add links in the description, add tags, and save it.

> Once you have an SLO set up, on the [Service Level Objectives status page][1] you can select an SLO from the list view to open its details side panel. The side panel will display the overall status percentage and remaining error budget for each of that SLO's targets, as well as status bars (monitor-based SLOs) or bar graphs (metric-based SLOs) of the SLI's history. If you have created a grouped monitor-based SLO using one [multi alert monitor][7] or a grouped metric-based SLO using the [`sum by` clause][8], in addition to the overall status percentage and remaining error budget you will also be presented with the status percentage and remaining error budget for each individual group.

Reviewer: ```suggestion
After you set up the SLO on the [Service Level Objectives status page][1], select an SLO from the list view to open the details side panel. The side panel displays the overall status percentage and remaining error budget for each of the SLO's targets, as well as status bars (monitor-based SLOs) or bar graphs (metric-based SLOs) of the SLI's history. If you created a grouped monitor-based SLO using one [multi alert monitor][7] or a grouped metric-based SLO using the [`sum by` clause][8], the status percentage and remaining error budget for each individual group is displayed in addition to the overall status percentage and remaining error budget.
```

> **Example:** If you create a monitor-based SLO to track latency per availability-zone, you will see status percentages and remaining error budget for the overall SLO and for each individual availability-zone that the SLO is tracking.

Reviewer: ```suggestion
**Example:** If you create a monitor-based SLO to track latency per availability-zone, the status percentages and remaining error budget for the overall SLO is displayed with each individual availability-zone that the SLO is tracking.
```

### Setting SLO targets

> To leverage the benefits of error budgets and Error Budget Alerts, you must set SLO target values strictly below 100%.

Reviewer: ```suggestion
To leverage the benefits of error budgets and error budget alerts, you must set SLO target values strictly below 100%.
```

> Setting a 100% target would mean having an error budget of 0% since error budget is equal to 100% - SLO Target. Without error budget representing acceptable risk, you face difficulty finding alignment between the conflicting priorities of maintaining customer-facing reliability and investing in feature development. In addition, SLOs with target values of 100% lead to division by zero errors in SLO Alert evaluation. 

Reviewer: ```suggestion
Setting a 100% target means having an error budget of 0% since error budget is equal to 100%—SLO target. Without error budget representing acceptable risk, you face difficulty finding alignment between the conflicting priorities of maintaining customer-facing reliability and investing in feature development. In addition, SLOs with target values of 100% lead to division by zero errors in SLO alert evaluation. 
```
